SYN-3238: Update certificates section to be more clear (#998)
splunk/public-o11y-docs: synthetics/test-config/private-locations.rst

> Adding certificates in Synthetics

Reviewer: ```suggestion
Add certificates in Synthetics
```

Author: Hi @gschatz-splunk! I can make this change, but it is pre-existing in docs: https://docs.splunk.com/observability/en/synthetics/test-config/private-locations.html#adding-certificates-in-synthetics

Would you still want me to proceed?

Reviewer: yep! I added it since i was in there and we had the opportunity to change it ;)

    docker run -e "RUNNER_TOKEN=<insert-token>" --volume=`pwd`/certs:/usr/local/share/ca-certificates/my_certs/ quay.io/signalfx/splunk-synthetics-runner:latest bash -c "sudo update-ca-certificates && bundle exec bin/start_runner"

> **Note for Browser tests:** When using custom root CA certificates, please be aware that they are not compatible with Browser tests. Browser tests require SSL/TLS validation for accurate testing. You do have the option to disable SSL/TLS validation for browser tests when necessary, but custom root CA certificates are not supported for this test type.

Reviewer: ```suggestion

.. Note:: Custom root CA certificates aren't supported for Browser tests. Browser tests require SSL/TLS validation for accurate testing. Optionally, you can deactivate SSL/TLS validation for Browser tests when necessary.

```
